### DISCUSSION TITLE
Sent Tokens are not showing up in my TCRO wallet

### DISCUSSION THREAD

**Discussion by Brownboy1995** [crypto-org-chain/chain-main#642] [Q&A]
I have request and received TCRO from the faucet as far as a successful transaction is concerned. Watching the crypto.org chain I've seen the sent tokens go live;however, when I check the balance of my wallet I do not see the complete transaction. Could anyone explain what is going on?

**Comment by aw126** [comment]
Hello @Brownboy1995, firstly we'd like to make sure if you are on the right faucet with the proper address. 
There are two different faucets - Croeseid Testnet Faucet & Cronos Testnet Faucet:
- Faucet for Croeseid: https://crypto.org/faucet (takes `tcro...` address)  
- Faucet for Cronos: https://cronos.crypto.org/faucet (takes `0x...` address)

Hope this solves your problem, please let us know if the problem persists.

**Comment by Brownboy1995** [comment]
Thank you.

 Yes, I have now requested tokens from Cronos to which it says the funds have been sent; however, do not show as deposited on my end. Nor have any of the funds from Croeiseid. 

I have each transaction number.

I

**Comment by lezzokafka** [comment]
Hello @Brownboy1995, may I ask how was the balance being checked? 
Was it by using `chain-maind q bank balances` or it was done by other means?

**Comment by Brownboy1995** [comment]
Yes, I had been making transactions on Croeseid Testnet initially to no avail.

The faucet initially was not working; however it began to. I see that the Cronos Faucet accepts ERC20 wallets which I had been made aware of by support; however, upon updating my App, the Cronos Testnet became available as opposed to the Cronos ERC20 being deposited. I began to make deposits to ERC20 and they showed up in Testnet upon the update, which to my understanding holds no financial value.

I made my business partner aware of the fact that I’ve been receiving trial/test funding finally; as well as have/had received the link to the CRONOS Faucet for funding. That was until I realized that only the CRONOS Testnet was available to me despite the address being ERC20.

With that being said, I am curious as to the actual funding as everything has a date/deadline.

I look forward to conducting this project on the EVM CRO blockchain. 

Please let me know what the next steps are in order to receive (funding) so that I may give my partner notice. 

Thanks

**Comment by Brownboy1995** [comment]
All of my CRONOS wallets now say no transactions when each one has been filled with TCRO. Now the CRONOS Faucet is no longer functioning properly. I believe Cassini may require the faucet to work and I understand CRONOS is becoming Mainnet this month, so I need these errors to be solved. 

Isaiah

**Reply by aw126** [reply]
Hello @Brownboy1995, thanks for reporting this. Yes, we are currently encountering some technical issues on the Cronos chain, and our team is performing upgrades on this. We will continue to provide the updates in [Discord](https://discord.gg/T2d9aAGw) and [Cassini GitHub](https://github.com/crypto-org-chain/cassini/blob/main/README.md) along with the progress.
